README.md

```diff
@@ -1,2 +1,401 @@
+
+### macOS
+
+1. Build projects directly from the command line
+
+      ```bash
+      cd ~/zxa
+      west build -p auto -b native_posix \
+        example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Run projects from the command line
+
+      ```bash
+      ./build/example-application/zephyr/projects/zxa_example_project/native_posix/zephyr/zephyr.exe
+      ```
+
+1. Build and run projects using Zephyr's `twister script`
+
+      ```bash
+      zephyr/scripts/twister \
+        -T example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Build and run the all projects (including projects and tests)
+   with code coverage using Zephyr's `twister script`
+
+      ```bash
+      zephyr/scripts/twister --coverage \
+        -T example-application/zephyr
+      ```
+
+### Windows
+
+1. Build projects directly from the command line
+
+      ```bat
+      cd %HOMEPATH%\zxa
+      west build -p auto -b native_posix \
+        example-application\zephyr\projects\zxa_example_project
+      ```
+
+1. Run projects from the command line
+
+      ```bat
+      .\build\example-application\zephyr\projects\zxa_example_project\native_posix\zephyr\zephyr.exe
+      ```
+
+1. Build and run projects using Zephyr's `twister script`
+
+      ```bat
+      zephyr\scripts\twister \
+        -T example-application\zephyr\projects\zxa_example_project
+      ```
+
+1. Build and run the all projects (including projects and tests)
+   with code coverage using Zephyr's `twister script`
+
+      ```bat
+      zephyr\scripts\twister --coverage \
+        -T example-application\zephyr
+      ```
+
+
+# Module Contribution Guidelines
+
+## Design Principles
+
+The ZXA module is designed around the following principles:
+
+- Keep all Zephyr-specific content within the (required) `zephyr` folder.
+
+  - This enables adding the example on to an existing repository.
+
+- Follow the Zephyr repository structure, mechanisms, and rules
+  whenever possible.
+
+  - This minimizes the variance from what users and developers expect,
+    simplifies maintenance, and reuses existing solutions.
+
+- Extend the Zephyr repository structure to support an arbitrary number
+  of applications and project configurations.
+
+  - The `projects` sub-tree identifies where non-test, non-sample projects
+    are expected to be found.  The `projects` folder is tied into the
+    Zephyr CI system through `module.yml`.
+
+- Maximize the amount of code automatically verifiable through Zephyr CI.
+
+- Minimize risk of conflicts in global namespaces through use of the
+  module abbreviation.
+
+  - Global namespaces exist in many areas, including the following areas.
+    Using the module abbreviation as a global symbol prefix reduces
+    the chance of collisions as other modules are created.
+
+    - Kconfig, CMake, C preprocessor symbols
+    - include file pathnames
+    - test and sample names
+    - shell commands
+    - settings identifiers
+
+
+## Example vs Sample
```

Comment:
I don't think we need this section either. This is way too much info, and it's a subjective statement.

Reply:
@carlescufi This is actually in here as a result of your question "example or sample"?  We have chosen to call it something different than "sample", so we need to define what the Zephyr Project means by "example".

Reply:
Removed.  We can work through the distinctions at a later time.